src/duckling/entitysystem/services/entity-box.service.ts

```diff
@@ -55,11 +59,35 @@ export class EntityBoxService extends BaseAttributeService<AttributeBoundingBox>
     }
 
     /**
+     * Set the entity's bounding box to a new value.
+     * @param  entityKey The entity to adjust the bounding box.
+     * @param  box Box the entity should be resized to.
+     * @param  mergeKey Pass to merge updates on the undo/redo stack.
+     */
+    setEntityBox(entity : Entity, box : Box2, mergeKey? : string) {
```

Comment:
Can you be explicit that this is returning an entity since the signature for these APIs are important?